packages/@aws-cdk-testing/framework-integ/test/aws-s3-assets/test/integ.assets.file-bundling.lit.ts

```diff
@@ -0,0 +1,39 @@
+import * as path from 'path';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { App, BundlingOutput, DockerImage, Stack, StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as assets from 'aws-cdk-lib/aws-s3-assets';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    /// !show
+    const asset = new assets.Asset(this, 'BundledAsset', {
+      path: path.join(__dirname, 'markdown-asset'), // /asset-input and working directory in the container
+      bundling: {
+        image: DockerImage.fromBuild(path.join(__dirname, 'alpine-markdown')), // Build an image
+        outputType: BundlingOutput.SINGLE_FILE, // Bundle only the file and do not package it
+        command: [
+          'sh', '-c', `
+            markdown index.md > /asset-output/index.html
+          `,
+        ],
+      },
+    });
+    /// !hide
+
+    const user = new iam.User(this, 'MyUser');
+    asset.grantRead(user);
+  }
+}
+
+const app = new App();
+const stack = new TestStack(app, 'cdk-integ-assets-bundling');
+
+new IntegTest(app, 'IntegTest', {
+  testCases: [stack],
+});
```

Comment:
Any chance you can add an assertions that checks the file is in the expected location and format, as opposed to being in a zip-archive?

Reply:
Might be easier to test that as a unit test though.

Reply:
Added a unit test